app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str):
+    name = command.split()
```

Comment:
You should check if the input string contains exactly three elements separated by spaces, with 'cp' as the first element, to ensure the command is valid. This is a requirement from the checklist.